tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "experimentalDecorators": true,
     "sourceMap": true,
     "importHelpers": true,
+    "importsNotUsedAsValues": "preserve",
```

Comment:
The ForkTsCheckerWebpackPlugin recommends setting this value.
See: https://www.npmjs.com/package/fork-ts-checker-webpack-plugin?activeTab=readme#type-only-modules-watching